test/run-verify-host-user.sh

```diff
@@ -185,7 +185,8 @@ done
 
 # execute run-tests
 test/common/run-tests --test-dir test/verify --nondestructive $exclude_options \
-    --machine localhost:22 --browser localhost:9090 $TESTS || RC=$?
+    --machine localhost:22 --browser localhost:9090 --track-naughties \
```

Comment:
This one we should drop, as that doesn't work in packit and downstream tests. It doesn't work there because these don't have a GitHub token. That will implicitly disable naughty updating, but it's confusing to explicitly enable it here (in vain).

Reply:
Ah thanks, I was not too sure about it, so went to the safest option. I will drop it.